.github/workflows/weekly-check-versions.yml

```diff
@@ -4,15 +4,39 @@ on:
   - cron: "0 6 * * 5"
 
 jobs:
-  stale:
+  create-pull-request:
     runs-on: ubuntu-latest
     steps:
-    - name: Run weekly
-      uses: actions/checkout@v2
-      with:
-       fetch-depth: '0'
-      
-    - name: Install dependencies
-      run: |
-        yarn install
-        yarn backstage-cli versions:bump
\ No newline at end of file
+      - uses: actions/checkout@v2
+        with:
+         fetch-depth: '0'
+
+      - name: Install dependencies and bump versions
+        run: |
+          yarn install
+          yarn backstage-cli versions:bump
+
+      - name: Create Pull Request
+        id: cpr
+        uses: peter-evans/create-pull-request@v3
+        with:
+          token: ${{ secrets.GITHUB_TOKEN  }}
+          commit-message: Update dependencies
+          committer: GitHub <noreply@github.com>
+          author: ${{ github.actor }} <${{ github.actor }}@users.noreply.github.com>
+          signoff: false
+          branch: update-dependecies
```

Comment:
Can we have concurrent PRs open with this ? perhaps we should put the date and perhaps time in the branch name?

Reply:
I am not sure about concurrent PRs, what would be use case for those? Yeah, that's good idea, I will put the date in branch name :) @iain-b

Reply:
I'm thinking we'll end up wanting to run this more regularly than once a week and perhaps we haven't merged a PR then the creation of the next PR fails as the branch already exists.

Reply:
Good point, I think it will just update the existing branch and treat that as one PR.